backends/dnf/pk-backend-dnf.c

```diff
@@ -3435,6 +3435,17 @@ pk_backend_upgrade_system_thread (PkBackendJob *job, GVariant *params, gpointer
 	/* set the installonly limit one higher than usual to avoid removing any kernels during system upgrades */
 	dnf_sack_set_installonly_limit (sack, dnf_context_get_installonly_limit (job_data->context) + 1);
 
+	/* reset libgit2 module when upgrading to F31: https://bugzilla.redhat.com/show_bug.cgi?id=1762751 */
+	if (g_strcmp0 (release_ver, "31") == 0) {
```

Comment:
Uhh, you guys? This patch will kick in for *any* distribution with the version 31. Why isn't this guarded behind a check on it being Fedora too?

Reply:
Sorry, not interested in supporting any other distros than Fedora and RHEL in this backend. Feel free to submit patches if you need.

Reply:
You shouldn't have merged this then. You *know* multiple distros use this backend, and we've historically not included distro-specific patches in *any* backend...

Reply:
Well, I suppose that's not completely true. We do have the distro-specific C files...